scripts/sct_label_utils.py

```diff
@@ -125,6 +125,10 @@ def process(self, type_process):
             self.output_image = self.continuous_vertebral_levels()
         if type_process == 'create-viewer':
             self.output_image = self.launch_sagittal_viewer(self.value)
+        if type_process == 'remove-specific':
```

Comment:
why not simply `remove` and `keep` for flags?

Reply:
The `remove` flag is already used as shown [here](https://github.com/neuropoly/spinalcordtoolbox/blob/4f22bd4176428763397840ce407fc0df74342e6d/scripts/sct_label_utils.py#L742) so the `-specific` part is simply for consistency between the two similar yet different flags.

Reply:
Would something such as `-delete` instead be preferable?

Reply:
ah! i see. I would suggest to rename this flag `remove-reference`, which I find more intuitive (although it will break retro-compatibility, but this feature is rarely used anyways, so no big deal)

Reply:
> Would something such as -delete instead be preferable?

no, let's keep the same terminology, but as suggested above, let's make it a `remove` and a `remove-reference`

Reply:
If we use `remove-reference`, should we keep `keep-specific` or change it to `keep-reference`?

Reply:
changes for `remove-reference` applied in commit [c158f65](https://github.com/neuropoly/spinalcordtoolbox/commit/c158f655d45706726d0e3173caf81ced17b1ac84)

Reply:
as mentioned here: https://github.com/neuropoly/spinalcordtoolbox/pull/2283#discussion_r288259468, the suggestion is to:
- rename remove --> remove-reference
- use remove for removing specified labels
- add keep for keeping specified labels

Reply:
Oops sorry I misunderstood! Newest changes in commit [bce701a](https://github.com/neuropoly/spinalcordtoolbox/commit/bce701a2843c85bf6686287dcdf5025df771929a)